docs/operations_tasks/add_entry_to_aws_secrets.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Add an entry to our secrets stored on AWS
+parent: Operations tasks
+---
+
+# Add an entry to our secrets that is stored on AWS
+The below steps assume that you have the correct account privileges in AWS.
+
+1. Log in to your HCA's AWS account
```

Comment:
```suggestion
1. Log in to your [HCA's AWS account](https://embl-ebi.awsapps.com/start#/)
```

Reply:
@amnonkhen That link is not working for me.